Step: build-step-credential-initializer-h76rm:

{"level":"warn","ts":1556772985.1804485,"logger":"fallback-logger","caller":"logging/config.go:65","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/backport-container-name-fix\" is not a valid GitHub commit ID"}
{"level":"info","ts":1556772985.182536,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}
Step: build-step-git-source-*********-test-qs-master-rrxhz:

{"level":"warn","ts":1556772986.5965106,"logger":"fallback-logger","caller":"logging/config.go:65","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/backport-container-name-fix\" is not a valid GitHub commit ID"}
{"level":"info","ts":1556772987.7749748,"logger":"fallback-logger","caller":"git-init/main.go:99","msg":"Successfully cloned \"https://github.com/*********/test-qs\" @ \"v0.0.18\" in path \"/workspace/source\""}
Step: build-step-place-tools:

Step: build-step-git-merge:

Warning failed to load version: failed to parse version -dev+9e73adc0b: No Major.Minor.Patch elements found
time="2019-05-02T04:56:31Z" level=info msg="got "
Using SHAs from PULL_REFS=master:
time="2019-05-02T04:56:31Z" level=warning msg="no SHAs to merge, falling back to initial cloned commit"
Step: build-step-setup-**-git-credentials:

Generated Git credentials file /workspace/xdg_config/git/credentials
Step: build-step-build-npm-install:

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN REPLACE_ME@1.0.0 No repository field.
npm WARN REPLACE_ME@1.0.0 No license field.

up to date in 0.404s
found 0 vulnerabilities

Step: build-step-build-npm-test:


> REPLACE_ME@1.0.0 test /workspace/source
> echo "no test specified" && exit 0

no test specified
Step: build-step-build-container-build:

[36mINFO[0m[0000] Downloading base image node:9-slim           
ERROR: logging before flag.Parse: E0502 04:56:41.034281       9 metadata.go:159] while reading 'google-dockercfg-url' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg-url
ERROR: logging before flag.Parse: E0502 04:56:41.038658       9 metadata.go:142] while reading 'google-dockercfg' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg
2019/05/02 04:56:41 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0002] Error while retrieving image from cache: getting file info: stat /workspace/sha256:0e0175bbe4f8d9d104590d8d2c591017874e30977cb71aaff1cf6fa99479575c: no such file or directory 
[36mINFO[0m[0002] Downloading base image node:9-slim           
2019/05/02 04:56:43 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0004] Executing 0 build triggers                   
[36mINFO[0m[0004] cmd: EXPOSE                                  
[36mINFO[0m[0004] Adding exposed port: 8080/tcp                
[36mINFO[0m[0004] Using files from context: [/workspace/source] 
[36mINFO[0m[0004] Skipping unpacking as no commands require it. 
[36mINFO[0m[0004] Taking snapshot of full filesystem...        
[36mINFO[0m[0004] ENV PORT 8080                                
[36mINFO[0m[0004] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0004] EXPOSE 8080                                  
[36mINFO[0m[0004] cmd: EXPOSE                                  
[36mINFO[0m[0004] Adding exposed port: 8080/tcp                
[36mINFO[0m[0004] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0004] WORKDIR /usr/src/app                         
[36mINFO[0m[0004] cmd: workdir                                 
[36mINFO[0m[0004] Changed working directory to /usr/src/app    
[36mINFO[0m[0004] Creating directory /usr/src/app              
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0004] Using files from context: [/workspace/source] 
[36mINFO[0m[0004] COPY . .                                     
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0004] CMD ["npm", "start"]                         
[36mINFO[0m[0004] No files changed in this command, skipping snapshotting. 
2019/05/02 04:56:47 existing blob: sha256:5bba3ecb4cd6374a19da45f2a3db0a9c0804984358b3bb4b666c28944016e5ff
2019/05/02 04:56:47 existing blob: sha256:7d083412657baa2dafba8d093034ef26f9ed2c90fb6ed95f76bb55aff03ed429
2019/05/02 04:56:47 existing blob: sha256:e01bd28a434b56ffd2a3fe9aa99b9b69aecca69904843edd30b1ee7150d79904
2019/05/02 04:56:47 existing blob: sha256:a88538b9cd05cfa611e43a25582a95ad7879f6fa708b0386a26b71ee62235bb3
2019/05/02 04:56:47 existing blob: sha256:196b8e3c919d5639193ba6c6e54774528942041bbc37606631a2250502985c3c
2019/05/02 04:56:51 pushed blob: sha256:6b269f0dd05b36fe481e17ed661fe00a31496de3bda673eda9a7fe2ed120b96c
2019/05/02 04:56:51 pushed blob: sha256:94e6dce150e0e69034ea259e1c45e46323c5e1f792974402824162cf53d887f4
2019/05/02 04:56:52 pushed blob: sha256:bc06cd11e3d69e45c79674987a23ae60578b86e60f86bc29147656a05d733875
2019/05/02 04:56:54 gcr.io/core-ui-dev/test-qs:0.0.18: digest: sha256:3f49c52fb9925567701494c9db87220ee843e634ee21181251e39889cc6eb802 size: 1404
Step: build-step-build-post-build:

no CVE provider running in the current ** namespace so skip adding image to be analysed
Step: build-step-promote-changelog:

Converted /workspace/source/charts/test-qs to an unshallow repository
Generating change log from git ref e109a35e0719abef14c078116c059e9b8884dd6a => abbad23afc8dbcb15fc373e5ec5e38655f2c758e
WARNING: Failed to enrich commits with issues: User.jenkins.io "" is invalid: metadata.name: Required value: name or generateName is required
WARNING: Failed to enrich commits with issues: User.jenkins.io "" is invalid: metadata.name: Required value: name or generateName is required
Finding issues in commit messages using git format
WARNING: No release found for *********/test-qs and tag v0.0.18 so creating a new release
Updated the release information at https://github.com/*********/test-qs/releases/tag/v0.0.18
generated: /workspace/source/charts/test-qs/templates/release.yaml
Created Release test-qs-0-0-18 resource in namespace **
Updating PipelineActivity *********-test-qs-master-17 with version 0.0.18
Updated PipelineActivities *********-test-qs-master-17 with release notes URL: https://github.com/*********/test-qs/releases/tag/v0.0.18
Step: build-step-promote-helm-release:

WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: chartmuseum.jenkins-x.io http://chartmuseum.jenkins-x.io
Successfully added Helm repository chartmuseum.jenkins-x.io.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Uploading chart file test-qs-0.0.18.tgz to http://jenkins-x-chartmuseum:8080/api/charts
Received 201 response: {"saved":true}
Step: build-step-promote-**-promote:

WARNING: prow based install so skip waiting for the merge of Pull Requests to go green as currently there is an issue with gettingstatuses from the PR, see https://github.com/jenkins-x/**/issues/2410
Promoting app test-qs version 0.0.18 to namespace **-staging
Created Pull Request: https://github.com/*********/environment-ivan-dev-staging/pull/20

WARNING: Failed to merge the Pull Request https://github.com/*********/environment-ivan-dev-staging/pull/20 due to PUT https://api.github.com/repos/*********/environment-ivan-dev-staging/pulls/20/merge: 405 Pull Request is not mergeable [] maybe I don't have karma?
Rebasing PullRequest due to conflict
Created Pull Request: https://github.com/*********/environment-ivan-dev-staging/pull/23

Pull Request https://github.com/*********/environment-ivan-dev-staging/pull/23 is merged at sha db4738dc11761ed887083049eaaeeec07921c513
Pull Request merged but we are not waiting for the update pipeline to complete!
